tests/tasks/test_juniperalarmtask.py

```diff
@@ -0,0 +1,140 @@
+from unittest.mock import patch
+
+import pytest
+
+from zino.config.models import PollDevice
+from zino.events import AlarmEvent, Events
+from zino.statemodels import DeviceStates
+from zino.tasks.juniperalarmtask import JuniperAlarmTask
+
+
+@pytest.mark.asyncio
+async def test_juniper_alarm_runs_without_errors(snmpsim, snmp_test_port):
+    device = PollDevice(
+        name="buick.lab.example.org",
+        address="127.0.0.1",
+        community="juniper-alarm",
+        port=snmp_test_port,
+    )
+    task = JuniperAlarmTask(device=device)
+
+    assert (await task.run()) is None
+
+
+@patch("zino.state.devices", DeviceStates())
+@pytest.mark.asyncio
+async def test_juniper_alarm_does_nothing_for_non_juniper_device(snmpsim, snmp_test_port):
+    from zino import state
```

Comment:
after #45, this should now be 
```suggestion
@patch("zino.state.state", ZinoState())
@pytest.mark.asyncio
async def test_juniper_alarm_does_nothing_for_non_juniper_device(snmpsim, snmp_test_port):
    from zino.state import state
```
 or something like that